<launch>
    <rosparam file="$(find hoverboard_driver)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find hoverboard_driver)/config/controllers.yaml" command="load"/>
    <node name="hoverboard_driver" pkg="ros2-hoverboard-driver" type="hoverboard_driver" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
        args="hoverboard_joint_publisher hoverboard_velocity_controller" />
</launch>
